### PR TITLE
refactor(log): unify log format with mx-apps

### PR DESCRIPTION
**Pull request by diqing-SHU**
[MX-1423](https://rewardops.atlassian.net/browse/MX-1423)

Minor change on log format to match requirement in ticket.

### CODE REVIEW

**Review by shamrt — CHANGES_REQUESTED** [review]
Huh. Seems we don't have a test to cover the actual log format in our test suite 😅 

Can you write one here? https://github.com/rewardops/rewardops-sdk-node/blob/master/test/utils/logger.test.js#L231-L296

**Comment by diqing-SHU** [comment]
> Huh. Seems we don't have a test to cover the actual log format in our test suite 😅
> 
> Can you write one here? https://github.com/rewardops/rewardops-sdk-node/blob/master/test/utils/logger.test.js#L231-L296

Good call! Will do.

File: test/utils/logger.test.js [hunk 1]
```diff
@@ -231,13 +231,22 @@ describe('#processLogData', () => {
...
   const mockLogLevel = faker.random.arrayElement(['error', 'warn', 'info', 'debug', 'verbose']);
+  const mockMessage = faker.lorem.sentence();
+
+  it('logs with correct format', () => {
+    const formattedLog = logFormat({ level: mockLogLevel, message: mockMessage, timestamp: mockTimestamp });
+
+    const expectedLog = `[${mockTimestamp}] [${LOG_PREFIX} ${mockLogLevel.toUpperCase()}] [${mockMessage}]`;
+
+    expect(formattedLog).toEqual(expectedLog);
+  });
```
Comment (shamrt) [inline test/utils/logger.test.js:242]: Optional: Can we drop this below the following test? I suppose they both have utility, but having this one above the subsequent test makes the latter seem slightly silly. I'm somewhat inclined to merge them, but perhaps not.

**Review by shamrt — APPROVED** [review]
Looks good!

**Review by Alex-Joseph — APPROVED** [review]
LGTM 👍🏻 

Nice work!

File: test/utils/logger.test.js (hunk 1 above)
Comment (diqing-SHU) [reply test/utils/logger.test.js:242]: Sure. I can do that

**Pull request merged**